<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="workers_list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>281</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_name">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_status">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Manager</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Employee</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Director</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="add_btn">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>170</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_btn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>170</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>remove</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
